### DISCUSSION TITLE
Unlock and rearrange cart/checkout blocks

### DISCUSSION THREAD

**Discussion by skinnerjoe** [woocommerce/woocommerce#43064] [Checkout Flow]
It would be nice to rearrange blocks like showing the billing address before the shipping address and remove blocks that are not needed such as the express checkout.

**Comment by senadir** [comment]
Express Checkout will not be visible if you don't have express payment methods enabled.

For switching between shipping and billing, can you tell us the use case exactly? as we have seem from research that putting shipping first converts better.

**Reply by skinnerjoe** [reply]
We only want to use the express payments on the product and cart pages, not the checkout page.
We prefer to have the billing address first because it is important to have this for our fraud protection as it's our number one problem. We then have the select box if you have a different shipping address. Having the shipping address first creates more AVS credit card declines as people pass over the fact that the billing address is needed for card verification. They are more apt to check the box to change to a different shipping address than they are the billing.
Moving blocks, changing sizes, or deleting unused fields seems to be the only reason to have blocks. Why have blocks if you cannot fully customize them? Woocommerce is preferred over Shopify for the ability to customize.

**Reply by skinnerjoe** [reply]
Hello,
It's been nearly two years and nothing has been done with this. Why won't WooCommerce allow us to reorder checkout fields? We are currently limited to using the classic checkout until this is permitted.

**Reply by jacs90** [reply]
> 
> For switching between shipping and billing, can you tell us the use case exactly? as we have seem from research that putting shipping first converts better.

For us 90% of our customers pickup from our winery. So having local pickup as some form of default either by having Billing address as the first option and having a shipping zone (Free Local Pickup) as default  or getting the local pickup section in  'woocommerce shipping' having the ability to be set as the default.

**Comment by JoshRyder** [comment]
We have customers that do not like PayPal due to censorship or threats of fines. Though we wish to offer PayPal Express Checkout as an option, many would prefer to see the regular checkout first. Our customers use cards 10/1 over PayPal and many have informed us they did not order because PayPal was right at the top or they did not see the other payment option right away.

**Reply by senadir** [reply]
You can disable express payment options for PayPal and just leave the regular gateway

**Reply by JoshRyder** [reply]
This is not the request. It would be nice to offer express checkout, just below the regular checkout.
